go.mod

```diff
@@ -3,107 +3,103 @@ module github.com/openshift/certman-operator
 go 1.17
 
 require (
-	github.com/Azure/azure-sdk-for-go v66.0.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.11.24
-	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
+	github.com/Azure/azure-sdk-for-go v63.1.0+incompatible
+	github.com/Azure/go-autorest/autorest v0.11.28
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.1
 	github.com/Azure/go-autorest/autorest/to v0.4.0
-	github.com/aws/aws-sdk-go v1.44.87
+	github.com/aws/aws-sdk-go v1.44.233
 	github.com/eggsampler/acme v1.0.0
-	github.com/go-logr/logr v1.2.2
-	github.com/lib/pq v1.10.6
-	github.com/openshift/api v0.0.0-20220531073726-6c4f186339a7
-	github.com/openshift/aws-account-operator/api v0.0.0-20220919121610-00e208e977e8
-	github.com/openshift/hive/apis v0.0.0-20220831193253-7dca200fad2f
-	github.com/openshift/operator-custom-metrics v0.5.1-0.20220802235640-dc76a1f15ee8
+	github.com/go-logr/logr v1.2.4
+	github.com/lib/pq v1.10.7
+	github.com/openshift/api v0.0.0-20230228142948-d170fcdc0fa6
+	github.com/openshift/aws-account-operator/api v0.0.0-20230322125717-5b5a00a3e99f
+	github.com/openshift/hive/apis v0.0.0-20230329164219-869f3fac05bd
+	github.com/openshift/operator-custom-metrics v0.5.0
```

Comment:
I double-checked this dep and I think it's ok for certman to go backwards here. Originally it was pinned to a commit to pick up this fix https://github.com/openshift/operator-custom-metrics/pull/44 like the rest of our hive operators, so they wouldn't need to explicity register routes during setup, however I see that it registers routes anyway https://github.com/openshift/certman-operator/blob/master/main.go#L79.

Reply:
Hmm good catch. Should we release a new version of operator-customer-metrics? cc @fahlmant

Reply:
Bumped to 0.5.1 to pick up all the latest changes